packages/node_modules/@webex/plugin-logger/test/unit/spec/logger.js

```diff
@@ -365,4 +365,51 @@ describe('plugin-logger', () => {
       });
     });
   });
+
+  describe('#walkAndFilter', () => {
+    it('redact Authorization', () => {
+      webex.config.logger.level = 'trace';
+      webex.logger.log({
+        Authorization: 'XXXXXXX',
+        Key: 'myKey'
+      });
+
+      // Assert auth was filtered
+      assert.calledWith(console.log, {Key: 'myKey'});
+
+      webex.logger.log({
+        authorization: 'XXXXXXX',
+        Key: 'myKey'
+      });
+
+      assert.calledWith(console.log, {Key: 'myKey'});
+    });
+
+    it('redact emails', () => {
+      webex.config.logger.level = 'trace';
+
+      // This behaviour does not seem correct
+      webex.logger.log('my email address is test@cisco.com');
+      assert.calledWith(console.log, 'my email address is test@cisco.com');
```

Comment:
Assumed the regex would filter all strings containing an email address. In practice it must be an exact match. Not sure that's correct but I'll leave it with the team as it's not the issue I'm addressing here.

Reply:
Thanks for catching this, we've added a JIRA to fix: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-140268